### PR TITLE
added ablity to display web app version

### PR DESCRIPTION
**Pull request by meltedspork**
So we can display version and keep track of it. I am fine moving it elsewhere tho.
screen shot 2018-02-01 at 7 30 56 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +32,8 @@
     "whatwg-fetch": "2.0.3"
   },
   "scripts": {
-    "start": "node scripts/start.js",
-    "build": "node scripts/build.js",
+    "start": "REACT_APP_VERSION=$npm_package_version node scripts/start.js",
```
Comment (kratsg) [inline package.json]: remove this change

File: package.json [hunk 2]
```diff
@@ -32,8 +32,8 @@
...
+    "build": "REACT_APP_VERSION=$npm_package_version node scripts/build.js",
```
Comment (kratsg) [inline package.json]: remove this change

File: scripts/build.js [hunk 3]
```diff
@@ -4,6 +4,8 @@
 process.env.BABEL_ENV = 'production';
 process.env.NODE_ENV = 'production';
 
+process.env.REACT_APP_VERSION = process.env.REACT_APP_VERSION || 'a.b.c';
```
Comment (kratsg) [inline scripts/build.js]: use `process.env.npm_package_version` which is a standard node env variable loaded from `package.json`

File: scripts/start.js [hunk 4]
```diff
@@ -4,6 +4,8 @@
 process.env.BABEL_ENV = process.env.BABEL_ENV || 'development';
 process.env.NODE_ENV = process.env.NODE_ENV || 'development';
 
+process.env.REACT_APP_VERSION = process.env.REACT_APP_VERSION || 'x.y.z';
```
Comment (kratsg) [inline scripts/start.js]: use `process.env.npm_package_version` which is a standard node env variable loaded from `package.json`

File: src/components/web-package-version.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+const navStyles = {
+  fontSize: '0.5em',
+  letterSpacing: '0.2em',
+};
+
+const WebPackageVersion = () => (
+  <sup>
+    <span className="badge badge-pill badge-secondary text-uppercase" style={navStyles}>
+      alpha {process.env.REACT_APP_VERSION}
```
Comment (kratsg) [inline src/components/web-package-version.jsx]: use `process.env.npm_package_version` which is a standard node env variable loaded from `package.json`

File: scripts/start.js (hunk 4 above)
Comment (meltedspork) [reply scripts/start.js:7]: This did not work, where did you find this information?

**Pull request merged**